### PR TITLE
+ 2 icons: Banxo, RATP

### PR DESCRIPTION
**Pull request by Ender-events**
# Description
Add 2 icons (Caisse d'epargne banxo, ratp)

### Added
Banxo (`com.caisseepargne.android.mobilebanking`)
Bonjour RATP (`com.fabernovel.ratp`)

## Contributor's checklist
- [x] I followed [the Lawnicons guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

File: svgs/caisseepargne_banxo.svg
Comment (x9136) [inline svgs/caisseepargne_banxo.svg:1]: - the icon size is too small;
- try to remove the fill by making 6x6px ellipse with 6px stroke width. The result is likely to be comparable to what was expected, so the fill can be avoided.

File: svgs/hello_ratp.svg
Comment (x9136) [inline svgs/hello_ratp.svg:1]: - the icon size is too small;
- replace the fill with 12px stroke.

File: app/assets/appfilter.xml
Comment (x9136) [inline app/assets/appfilter.xml:1]: drawable="caisseepargne_banxo" → drawable="banxo" 
caisseepargne_banxo.svg → banxo.svgs

drawable="hello_ratp" → drawable="bonjour_ratp"
hello_ratp.svg → bonjour_ratp.svgs

**Review by x9136 — CHANGES_REQUESTED** [review]

**Comment by Ender-events** [comment]
If #2159 is accepted, you can close this PR

**Comment by x9136** [comment]
#2174

**Pull request closed without merging**